docs/UserGuide.md

```diff
@@ -313,26 +314,31 @@ Format: `clear`
 
 ### Importing data : `import`
 
-Imports a .csv file compatible with EmployeeManager into the application. It will overwrite the existing
-data stored in the application.
+Imports an EmployeeManager data file into the application.
 
 * The file path must be valid, and the application must have permission to read from the file.
+* No changes will be made if the data file is invalid or not found.
+
+<div markdown="span" class="alert alert-danger">:exclamation: **Danger:**
+The import command will **overwrite** existing data in the application. If you wish to keep a copy of the existing
+data, it is recommended to use the `export` command to save a copy of the current data.
+</div>
 
 Format: `import FILE_PATH`
 
 Examples:
-* `import C:\Data\EmployeeList.csv` imports the .csv file from the specified path.
+* `import ./data.json` imports the data in the data.json data file.
```

Comment:
Should we just use FILE_NAME instead as users might not know what FILE_PATH is. For example, we can just fix the prefix "./" in front of the FILE_NAME. so for format it can be something like this,
Format: `import ./FILE_NAME`
Requirement: The file with the indicated FILE_NAME must be in the same folder as EmployeeManager

Reply:
That was a consideration I had, but I felt having the flexibility would be more convenient (not having to drag the data file in and out of folders).

I am planning to add a GUI way to import/export to support non-technical users. So the CLI will allow the data file to be imported/exported from any valid path and the GUI will allow those who don't know how to get the path to use the commands as well. Would this be a good idea?

Reply:
Ohh in that case, that should be fine. Also, could you remove the "data in", so the sentence will just be
`import ./data.json` imports the data.json data file.
That makes it sound more readable

Reply:
Okay, I have made the changes.